content/docs/command-reference/checkout.md

```diff
@@ -1,7 +1,7 @@
 # checkout
 
 Update data files and directories in the <abbr>workspace</abbr> based on current
-DVC-files.
+`dvc.lock` and `.dvc` files.
```

Comment:
- [ ] Need to figure out a better way to refer to these so we don't have to repeat "dvc.yaml and .dvc files" or "dvc.lock and .dvc files" so often... Ivan suggests a new `<abbr>DVC files</abbr>` for now... Will get back to this 🧠 (but maybe extract it from this PR.

Reply:
Created iterative/dvc/issues/4278 about this for now...